src/VSTS-Bot.Api/Dialogs/ConnectDialog.cs

```diff
@@ -215,6 +215,14 @@ public ConnectDialog(string appId, string appScope, Uri authorizeUrl, IAuthentic
                 .OrderBy(a => a)
                 .ToList();
 
+            if (!accounts.Any())
+            {
+                reply.Text = Exceptions.NoAccounts;
```

Comment:
Why do we store normal text response in the Exceptions resource? Looks a little bit inconsistent.